module/zfs/arc.c

```diff
@@ -7313,6 +7337,7 @@ arc_init(void)
 
 	/* Convert seconds to clock ticks */
 	arc_min_prefetch_lifespan = 1 * hz;
+	arc_min_long_lifespan = 6 * hz;
```

Comment:
Kind of arbitrary, but 5s for a txg + 1s of buffer time. `ARC_FLAG_LONG_LIFE` was actually added before I corrected a problem with how the new prefetcher interacts with the ARC. The problem caused the MFU side of the ARC to become artificially dominant, meaning newly prefetched buffers were being evicted before they could be used by the scanning code. This flag may not be necessary anymore. I will run some tests and remove it if it is not.